tests/integration/start_codis.sh

```
@@ -21,16 +36,31 @@ mkdir codis_data_1
+
+# Start instances one by one with time to initialize
 ./pika -c ./pika_8000.conf
+sleep 5
 ./pika -c ./pika_8001.conf
 #ensure both master and slave are ready
 sleep 10
 
+# Display memory usage after starting pika instances
+free -h || true
+
 cd ../codis
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)